Mathlib/NumberTheory/RamificationInertia/Basic.lean

```diff
@@ -238,6 +239,23 @@ lemma inertiaDeg_map_eq [p.IsMaximal] (P : Ideal S)
 
 end DecEq
 
+
+section absNorm
+
+/-- The absolute norm of an ideal `P` above a rational prime `p` is
```

Comment:
Do we have the same result for an arbitrary extension of global fields?

Reply:
I don't think so (I am 99.9% sure). Anyway this result would not be a special case, since `absNorm` is a number, and `relNorm` is an ideal.

Reply:
No, no, I agree; but the proof would probably be similar, so mine was a "soft encouragement" to try to add that result as well 😄

Reply:
I tried, but since `relNorm` is defined as the ideal generated by the norm of all elements it requires more work and it's probably suitable for another PR.